src/components/PromoBanner/PromoBanner.unit.spec.jsx

```diff
@@ -74,4 +74,40 @@ describe('<PromoBanner>', () => {
       ).to.be.true;
     });
   });
+
+  describe('onClose', () => {
+    let wrapper;
+    const onCloseSpy = sinon.spy();
+    const props = {
+      type: PROMO_BANNER_TYPES.announcement,
+      href: 'https://missionact.va.gov/',
+      text: TEXT,
+      onClose: onCloseSpy,
+    };
+
+    beforeEach(() => {
+      wrapper = shallow(<PromoBanner {...props} />);
+    });
+
+    afterEach(() => {
+      onCloseSpy.resetHistory();
+      wrapper.unmount();
+    });
```

Comment:
I'd recommend defining a function which returns a new wrapper and spy. These hooks can get funky.
```js
const renderBanner = () => {
  const onCloseSpy = sinon.spy();
  const wrapper = shallow(
    <PromoBanner
        type={PROMO_BANNER_TYPES.announcement}
        href="https://missionact.va.gov/"
        text={TEXT}
        onClose={onCloseSpy}
    />
  );
  return { onCloseSpy, wrapper };
};

it('...', () => {
  const { wrapper, onCloseSpy } = renderBanner();
  ...
});
```